src/admin/redux/modules/reset_payments.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+import { createAction } from 'redux-actions'
+import type { Action, Reducer } from 'redux'
+import { get_body } from 'app/http'
+
+const RESET_SUBSCRIPTION_PAYMENTS =
+  'RESET_SUBSCRIPTION_PAYMENTS'
+
+const initialState = {}
+type State = typeof initialState
+
+const reducer: Reducer<State, Action> =
```

Comment:
In that case we need a reducer. But we shouldn't really add the reducer til we need it